tools/reactive-controllers/src/LanguageResolution.ts

```diff
@@ -19,7 +19,18 @@ export const languageResolverUpdatedSymbol = Symbol(
 
 export class LanguageResolutionController implements ReactiveController {
     private host: ReactiveElement;
-    language = document.documentElement.lang || navigator.language;
+    get language(): string {
+        return document.documentElement.lang || navigator.language;
```

Comment:
This is fetching the root-level language but shouldn't we also be honoring if the language attribute is set on a container too?

Reply:
this change is what causes a majority of the number field failing tests, I would like to revert it since its out of scope for this specifically and capture this idea somehow for when we refactor number field.

Reply:
ok looks like you made some edits since i started and ill retest

Reply:
Those tests were failing before as well - I added this change to resolve the first round of issues I saw. You can see the details outlined in the Jira ticket.

> The first error I identified was on NumberField which threw an error that locale was invalid on every instance. I fixed this according to similar debugging advice I found online by converting any manually provided locales with underscores to dashes.

Reply:
thats odd, when i reverted it the tests passed. I'm going to do a fresh clone again to make sure it wasn't something weird on my local. thank you for this context and apologies I missed that from the ticket.